src/lenses/CrvRefundLens.sol

```diff
@@ -146,8 +145,11 @@ contract CrvRefundLens {
         }
 
         if (totalFeesWithdrawn > refund) {
-            // IBentoBoxV1(DEGENBOX).withdraw(MIM, msg.sender, msg.sender, 0, totalFeesWithdrawn);
-            // MIM.transfer(FEE_WITHDRAWER, totalFeesWithdrawn - refund);
+            console2.log("Transfer: ", totalFeesWithdrawn - refund);
+            console2.log("Sender: ", msg.sender);
+            console2.log("Total fees: ", totalFeesWithdrawn);
+            IBentoBoxV1(DEGENBOX).withdraw(MIM, msg.sender, msg.sender, 0, totalFeesWithdrawn);
```

Comment:
This line keeps throwing the error `Reason: BentoBox: no masterContract` but I cannot, for the life of me, figure out why.

I believe it's throwing here... I am, however, using pushPrank(OPS_MULTISIG) and because the from arg is msg.sender, I thought it should just auto-pass the allowed modifier. What am I doing wrong?

https://etherscan.io/address/0xd96f48665a1410C0cd669A88898ecA36B9Fc2cce#code#F1#L749